ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -75,11 +77,78 @@ + (BOOL)validBrokerRedirectUri:(NSString*)url
     return NO;
 }
 
-+ (void)internalHandleBrokerResponse:(NSURL *)response
++ (BOOL)internalHandleBrokerResponse:(NSURL *)response
 {
 #if TARGET_OS_IPHONE
-    ADAuthenticationCallback completionBlock = [ADBrokerHelper copyAndClearCompletionBlock];
-    HANDLE_ARGUMENT(response, nil);
+    __block ADAuthenticationCallback completionBlock = [ADBrokerHelper copyAndClearCompletionBlock];
+        
+    ADAuthenticationResult* result = [self processBrokerResponse:response];
+    
+    [[NSUserDefaults standardUserDefaults] removeObjectForKey:kAdalResumeDictionaryKey];
```

Comment:
do we need to remove the flag in function callbackCleanup of ADBrokerNotificationManager.m too?